packages/panzer/disc-fe/src/Panzer_L2Projection_impl.hpp

```diff
@@ -258,25 +259,34 @@ namespace panzer {
     }
     PHX::exec_space::fence();
 
-    ghostedMatrix->fillComplete();
-    const auto exporter = factory.getGhostedExport(0);
-    ownedMatrix->doExport(*ghostedMatrix, *exporter, Tpetra::ADD);
-    ownedMatrix->fillComplete();
-
+    {
+      PANZER_FUNC_TIME_MONITOR("Exporting of mass matrix");
+      ghostedMatrix->fillComplete();
```

Comment:
If you're calling fillComplete on the ghosted matrix, you need to provide the correct domain and range Maps.  This is because the ghosted matrix's row Map is not one to one.

Reply:
@rppawlo can you address @mhoemmen comments?  Just changed the indentation...